user/wpaperd/template.py

```diff
@@ -0,0 +1,34 @@
+pkgname = "wpaperd"
+pkgver = "1.0.1"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = [
+    "cargo-auditable",
+    "pkgconf",
+    "scdoc",
+]
+makedepends = ["mesa-devel", "rust-std", "wayland-devel"]
+pkgdesc = "Modern wallpaper daemon for Wayland"
+maintainer = "tulilirockz <tulilirockz@outlook.com>"
+license = "GPL-3.0-or-later"
+url = "https://github.com/danyspin97/wpaperd"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "e0e4d8ef16ae5b48fbbfd7c280c0029974fc96352811b56e3a3dbf37ef67db83"
+# check: no meaningful tests
+options = ["!check"]
+
+
+def install(self):
+    self.install_bin(f"target/{self.profile().triplet}/release/wpaperd")
+    self.install_bin(f"target/{self.profile().triplet}/release/wpaperctl")
+
+
+def post_build(self):
+    with open(f"{self.cwd}/man/wpaperd-output.5.scd", "rb") as i:
+        with open(f"{self.cwd}/wpaperd-output.5", "w") as o:
+            self.do("scdoc", input=i.read(), stdout=o)
```

Comment:
these are normally visually ordered in the order they run, so post_build goes above install in the template